### PR TITLE
Windows updates APIs: Add unknownFutureValue to enums

### PR DESCRIPTION
**Pull request by Alice-at-Microsoft**

### CODE REVIEW

**Comment by Alice-at-Microsoft** [comment]
@isvargasmsft Adding "unknownFutureValue" as we discussed

File: api-reference/beta/resources/enums-windowsupdates.md [hunk 1]
```diff
@@ -28,6 +29,7 @@ Namespace: microsoft.graph.windowsUpdates
 |offeringByRequest|
 |pausedByRequest|
 |pausedByMonitoring|
+|unknownFutureValue|
 |faultedByContentOutdated|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/enums-windowsupdates.md:33]: This item has values after the unknownFutureValue, this means we should clarify to add the proper headers in the property description of all resources that use this enum. Guidance here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/174164/Evolvable-enumerations?anchor=step-3%3A-add-clarification-if-one-or-more-members-follow-unknownfuturevalue

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Let's check the description of properties that use these enums include `unknownFutureValue` and for those with values after this one, the proper mention about the headers.

**Comment by Alice-at-Microsoft** [comment]
Thanks, @isvargasmsft! I knew I was missing something...

Some notes based on how I interpreted the instructions:
* I included `unknownFutureValue` even in property descriptions for read-only properties (i.e. even though the value is [never accepted in a request](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103144/Evolvable-enums?anchor=requests))
* I included `unknownFutureValue` in property descriptions for properties that only handle a subset of the enum values in the schema

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**